### ISSUE TITLE
Impact of removing PJS from Firefox

### ISSUE THREAD

**Issue by southerngs** [IntelLabs/RiverTrail#46]
I noticed that Parallel Javascript has been removed from Firefox.  This is described in [Bug 1117724](https://bugzilla.mozilla.org/show_bug.cgi?id=1117724).  As a result I think that RiverTrail doesn't work for newer versions of Firefox (it worked for me with my release version 35.0, but not when I was testing with the nightly build 38.0a1 (2015-01-15).

I'm wondering what the impact removing PJS from Firefox has on the RiverTrail project.

**Comment by jsreeram** [comment]
@southerngs , the River Trail prototype (this repo) and the PJS implementation in Nightly are two different, independent implementations. So River Trail will continue to work in Firefox.

**Comment by southerngs** [comment]
@jsreeram Thanks for the reply.  I noticed that when I ran some of the demos that they work with Firefox 35.0, but not with the Firefox Nightly 38.0a1.  But I just started looking at this today.  Should I expect River Trail to work with the Nightly versions of Firefox?

Also for references the demos that I noticed this with are: 
[NBody](http://intellabs.github.io/RiverTrail/examples/nbody-webgl/NBody.html) and that the [interactive shell](http://intellabs.github.io/RiverTrail-interactive/) reports that it works in parallel mode for Firefox 35, but only in serial mode with 38.0a1.

Also I'm seeing the same behavior with both OS X and with Linux.
